Prototope/Animation.swift

```diff
@@ -48,6 +48,8 @@ public class LayerAnimatorStore {
 	public var backgroundColor: Animator<Color>
 	public var alpha: Animator<Double>
 	public var rotationRadians: Animator<Double>
+    //kPOPScaleXY expects two values, not one!
+    public var scale: Animator<Point>
```

Comment:
Hm, that’s GIthub’s rendering messing it up - I use spaces and you used tabs, it seems :)